### ISSUE TITLE
Deprecation: 

### ISSUE THREAD

**Issue by remtav** [NRCan/geo-deep-learning#220]
Features from CoordConv and addind spatial resolution a training band (#82) are still present in GDL. Have they been used lately? Maybe time for deprecation.

**Comment by mpelchat04** [comment]
Agreed. These have not been used recently and the tests we did back then with these features were not suggesting it would improve results.   

This relates to CoordConv, spatial resolution as training band and CanvecAux data.
